paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -92,6 +116,13 @@ class CustomerSheet @Inject internal constructor(
      * a persisted payment option selection.
      */
     suspend fun retrievePaymentOptionSelection(): CustomerSheetResult = coroutineScope {
+        val request = configureRequest
+            ?: return@coroutineScope CustomerSheetResult.Failed(
+                IllegalStateException(
+                    "Must call `configure` first before attempting to fetch the saved payment option!"
```

Comment:
Should we move this out of the `coroutineScope`? Might give better stack traces to debug/fix.